docs/Configurations.md

```diff
@@ -1,99 +1,86 @@
 # FCA: Configurations
 
-TODO: add intro
+This document gives the various configurations available for Firebolt Certification App (FCA). It provides insights into setting up and utiizing different environments, parameters, and app types to facilitate effective testing and certification of applications using firebolt technology. 
+
 
 ## Table of Contents
 
-- [Supported Targets](#supported-targets)
+- [Running with MFOS](#running-with-mfos)
 - [Connect to Mock Firebolt](#connect-to-mock-firebolt)
 - [Supported URL Parameters](#supported-url-parameters)
 - [App Types](#app-types)
 
-## Supported targets
-
-EXISTING CONTENTS(needs revision):
-> - MFOS - `https://github.com/rdkcentral/mock-firebolt`
+## Running with MFOS
 
 ### Background
 
-TODO: add details
+Mock Firebolt (MFOS) is a simulated environment that FCA can interact with.
 
 ### Setup
 
-TODO: add details
+- *MFOS*: Accessible at [Mock-Firebolt](https://github.com/rdkcentral/mock-firebolt)
+- Use Mock Firebolt's [usage documentation](https://github.com/rdkcentral/mock-firebolt#usage-local) to get set up. 
 
 ### Usage
 
-TODO: add details
+Utilize MFOS for testing in a controlled, simulated environment. MFOS can be ideal for early-stage development and testing.
 
 ## Connect to Mock Firebolt
```

Comment:
We don't need two sections, `Running with MFOS` and `Connect to Mock Firebolt`, they should be combined into one.

Reply:
merged both 
currently looking like this :
in TOC:
## Table of Contents

- [Running with MFOS](#running-with-mfos)
  - [How to Connect with Mock Firebolt?](#connect-to-mock-firebolt)
  
  
  
  ## Running with MFOS
  ### Connect to Mock Firebolt

Reply:
Just one section for MFOS -`How to Connect with Mock Firebolt?` is just setup/usage instructions for the MFOS section.

Reply:
Looks much better. A few things
- Comment from above on step by step instructions on how to use FCA with MFOS.
- Change the TOC link to match the section title.
- Think `Use MFOS if you don't have a device with firebolt running on it, or don't have a device at all.` should go in background.

Reply:
updated to this:
## Connect to Mock Firebolt

### Background

Mock Firebolt (MFOS) is a simulated environment (representing real Firebolt running on device) that FCA can interact with.
Connecting to Mock Firebolt enables testing against a simulated Firebolt OS, providing a safe and flexible enviroment for development and testing.
Use MFOS if you don't have a device with firebolt running on it, or don't have a device at all.

### Setup

- *MFOS*: The open source GitHub project is accessible at [Mock-Firebolt](https://github.com/rdkcentral/mock-firebolt)
- Use Mock Firebolt's [usage documentation](https://github.com/rdkcentral/mock-firebolt#usage-local) to get set up. 
- Clone the Repository [Mock-Firebolt](https://github.com/rdkcentral/mock-firebolt)
- Follow the instructions provided in the repository to start the Mock Firebolt.
- Launch the Firebolt Certification App and add `mf=true` to the app launch URL to connect to the mock environment, visit the [URL Parameters](#supported-url-parameters) for details on sending Parameters.

### Usage

Utilize MFOS for testing API integrations and app behavior in a controlled, simulated environment. MFOS can be ideal for early-stage development and testing.
On FCA side only passing `mf=true` URL parameter is required. For detailed instructions on the setup and usage check [MFOS Usage](https://github.com/rdkcentral/mock-firebolt?tab=readme-ov-file#usage-local).